Dockerfile

```diff
@@ -68,8 +68,7 @@ RUN oioioi-create-config /sio2/deployment
 WORKDIR /sio2/deployment
 
 RUN sed -i -e \
```

Comment:
I've already started working towards cleaning up this part in #134, however now that I think about it, it would probably be better for maintenance if the settings file doesn't get any additional modifications in the process of building containers (ie. less mental load on maintainers who have to remember to ensure any changes to the settings in the `Dockerfile` aren't breaking anything in the core settings).

Reply:
I think the environment variables approach is the most flexible, as it will fit well with non-docker installations.
I already updated worflow in #160

Reply:
Thanks, based on deletions in #134 I assume that this fragment of Dockerfile mostly redundant. It's gonna save some time